src/libpriv/rpmostree-core.cxx

```diff
@@ -297,6 +297,11 @@ rpmostree_context_new_compose (int               userroot_dfd,
 
   rpmostree_context_set_cache_root (ret, userroot_dfd);
 
+  auto platform_module = treefile_rs.get_platform_module();
+  if (!platform_module.empty()) {
```

Comment:
Side note: the code here is GNU style, which has the opening brace on a new line.  But honestly, the real fix is to slowly drain the C/C++ into Rust which made the correct decision in having a standard out of the box code formatter :smile: 

So, not worth a respin.  But please do keep this in mind for larger future changes.

Reply:
(Maybe we should investigate `clang-format` or something but I am doubtful it's worth it)

Reply:
Sorry, as a former FreeBSD committer I'm hard-wired into their C code style and can never escape. 🙃